include/libdnf/module/module_dependency.hpp

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef LIBDNF_MODULE_MODULE_DEPENDENCY_HPP
+#define LIBDNF_MODULE_MODULE_DEPENDENCY_HPP
+
+#include <string>
+#include <vector>
+
+namespace libdnf::module {
+
+
+class ModuleDependency {
+public:
+    ModuleDependency(std::string module_name, std::vector<std::string> streams)
+        : module_name(module_name),
+          streams(streams) {}
+
+    /// @return Name of the required module.
+    /// @since 5.0
+    const std::string get_module_name() const { return module_name; };
+
+    /// @return Vector of streams. Prefix '-' denotes conflicting stream, otherwise, the stream is one of required.
+    ///         If there are no other streams, any active stream of the module can satisfy the require.
+    /// @since 5.0
+    const std::vector<std::string> get_streams() const { return streams; };
+
```

Comment:
What about to `add to_string` method? - then we do not need in ModuleItem get_dependencies that will return vector<string>.

Reply:
It can be useful for outputs like `dnf module info` and it can be internally used by get_module_dependencies_string()